bw2h5.py

```diff
@@ -10,28 +10,36 @@
 # license agreement from NVIDIA CORPORATION is strictly prohibited.
 #
 
-"""Read data from bigWig files in intervals and generate batch data for model.
+r"""Read data from bigWig files in intervals and generate batch data for model.
 
 Workflow:
     1. Reads a BED file containing genomic intervals
     2. Takes as input a bigWig file containing noisy ATAC-Seq data
     3. Optionally, selects intervals with nonzero coverage in the noisy data
     4. Splits intervals into batches of given size
-    5. Writes each batch to an hdf5 file
-    6. Optionally, includes clean data and clean peaks
+    5. Reads noisy ATAC-seq data in these intervals from the bigWig file
+    6. Optionally, includes other layers of input in addition to the noisy
+       ATAC-seq data
+    7. Writes each batch to an hdf5 file
+    8. Optionally, includes clean data (regression labels) and clean peaks
+       (classification labels)
 
 Output:
-    h5 file containing data for training or testing DL model
+    .h5 file containing data for training or testing a DL model
 
 Examples:
     Training:
-        python bw2h5.py --noisybw noisy.bw --intervals training_intervals.bed
-            --batch_size 120 --prefix training_data
+        python bw2h5.py --noisybw noisy.bw --intervals training_intervals.bed \
+            --batch_size 2000 --prefix training_data \
```

Comment:
120 seemed unnecessarily small. But I've deleted the batch_size argument in the example commands now. Since I've set the default to 1000, there should be no need to set this manually.